.travis.yml

```
@@ -121,20 +121,52 @@ matrix:
         # We use manual clone, with depth and single branch = the fastest
         # Because of this, we need to create .mbed file as the current tools require it
         - >-
-         git clone --depth=1 --single-branch --branch feature-cmake https://github.com/ARMmbed/mbed-os-example-blinky.git;
-         cd mbed-os-example-blinky;
+         git clone --depth=1 --single-branch --branch feature-cmake https://github.com/ARMmbed/${EXAMPLE_NAME}.git;
+         cd ${EXAMPLE_NAME};
          git clone --depth=1 --single-branch https://github.com/ARMmbed/mbed-os.git;
          cd mbed-os;
          git fetch origin +refs/pull/${TRAVIS_PULL_REQUEST}/merge;
          git checkout -qf FETCH_HEAD
         - >-
           cd -;
           touch .mbed;
-          mbedtools configure -t GCC_ARM -m K64F;
+          mbedtools configure -t GCC_ARM -m ${TARGET_NAME};
           mkdir -p build
       script:
         - cd build && cmake .. -GNinja -DMBED_PROFILE=develop && cmake --build .
 
+    - <<: *cmake-build-test
+      name: "CMake kvstore example"
+      env: NAME=cmake_test EXAMPLE_NAME=mbed-os-example-kvstore TARGET_NAME=K64F
+
+    - <<: *cmake-build-test
+      name: "CMake ble example"
+      env: NAME=cmake_test EXAMPLE_NAME=mbed-os-example-ble TARGET_NAME=K64F
```

I think there is only one example sub-project within this repository that has been enabled.
```suggestion
      env: NAME=cmake_test EXAMPLE_NAME=mbed-os-example-ble TARGET_NAME=NRF52840_DK
```
It should use the target `NRF52840_DK` as it has the most up-to-date BLE hardware.